Tests/VaporTests/ApplicationTests.swift

```diff
@@ -78,11 +78,31 @@ final class ApplicationTests: XCTestCase {
         let request = Request(application: app, on: app.eventLoopGroup.next())
         request.headers.contentType = .json
         request.url.path = "/foo"
-        print(request.url.string)
         request.url.query = "hello=world"
-        print(request.url.string)
         try XCTAssertEqual(request.query.get(String.self, at: "hello"), "world")
     }
+    
+    func testWrappedSingleValueQueryDecoding() throws {
```

Comment:
Could you add a comment here with a link to the PR for future reference?
```suggestion
    // https://github.com/vapor/vapor/pull/2163
    func testWrappedSingleValueQueryDecoding() throws {
```